lib/data_completeness.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# The completeness of each type of member data as a percentage
+class DataCompleteness
+  # @param [Array<PersonCard>]
+  def initialize(person_cards:)
+    @person_cards = person_cards
+  end
+
+  CARDS = %i(social bio contacts identifiers).freeze
+  Percentages = Struct.new(*CARDS)
+  # The percentage of data completeness for categories defined in CARDS
+  # @return [struct DataCompleteness::Percentages]
```

Comment:
Did you see somewhere that you should be including this `struct` here? I can't see anything in http://www.rubydoc.info/gems/yard/file/docs/Tags.md about this, though I could easily be overlooking something